cpp/cmake-library/custom-publication-plugin/.gitignore

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Ah, I think this isn't supposed to be checked in here. It should be only generated into a repo and used as a source dependency.